pkgs/servers/home-assistant/custom-components/dreame-vacuum/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  python312Packages,
```

Never consume `python312Packages` in `home-assistant-custom-components`. You must consume the same package set that home-assistant provides. To that end you can just consume the dependencies up here.

```suggestion
  numpy,
  pillow,
  ...,
```

And please sort this list.